### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by juliamourac**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs update

* **What is the current behavior?** (You can also link to an open issue here)
https://github.com/dojot/history/blob/development/README.md

* **What is the new behavior (if this is a feature change)?**
* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
dojot/dojot#1210

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,124 +1,89 @@
...
+In order to properly start History, the following variables must be set. All
+values are the default ones, and they need to be modified to the services' address.
+
+Key                         | Purpose                                                      | Default Value
+----------------------------|--------------------------------------------------------------|----------------
+HISTORY_DB_ADDRESS          |History database's address                                    |"mongodb"
+HISTORY_DB_PORT             |History database's port                                       |27017
+HISTORY_DB_REPLICA_SET      |History database's replica set address                        |None
+HISTORY_DB_DATA_EXPIRATION  |Seconds before removing an entry in MongoDB                   |604800
+MONGO_SHARD                 |                                                              |False
```
Comment (raulnegreiros) [inline README.md]: Missing the purpose

File: README.md [hunk 2]
```diff
@@ -1,124 +1,89 @@
...
-installed within a virtual environment. This can be created by:
+### Standalone setup
+
+As this service is written in **Python 2.7**, it is recommended that all packages are
+installed within a [virtual environment](https://github.com/pypa/virtualenv). It can be created by:
 
 ```shell
-virtualenv ./venv
-source ./venv/bin/activate
+$ virtualenv ./venv
```
Comment (raulnegreiros) [inline README.md]: `$` is not necessary once the file reader will format it as commands, in this way the user can copy and paste the content without modifications

File: README.md [hunk 3]
```diff
@@ -1,124 +1,89 @@
 # dojot History service
```
Comment (raulnegreiros) [inline README.md]: The title is about the history service, but the content is about the `history` and `persister`. Isn't clear what is the role of each one.

You could modify the document structure in order to improve the understanding. I suggest something like:
A brief description talking about the repository itself.
A section exclusively to the `history`
A section exclusively to the `persister`

The service's section with:
- Description
- Configuration
- Environment variables
- Installation (stand-alone and container case)
- How to run (stand-alone and container case)
- API

Remember: the services are together, but they are not the same service.

File: README.md [hunk 4]
```diff
@@ -131,7 +96,9 @@ For History service, just set all needed environment variables and execute:
 
 Persister is much simpler, just execute:
 ```bash
-python -m history.subscriber.kafkaSubscriber
+$ python -m history.subscriber.kafkaSubscriber
+#if the above command doesn't work, try:
```
Comment (raulnegreiros) [inline README.md]: Why it could not work? :sweat_smile:
It's the user instructions, must be assertive :dart:

File: README.md [hunk 5]
```diff
@@ -12,24 +12,116 @@ employing different filters, like deviceId, number of entries to be listed and s
...
+HISTORY_DB_REPLICA_SET      |History database's replica set address                        |None
+HISTORY_DB_DATA_EXPIRATION  |Seconds before removing an entry in MongoDB                   |604800
+MONGO_SHARD                 |Activate the use of sharding or not                           |False
+AUTH_URL                    |Auth url address                                              |"http://auth:5000"
+
+### How to install
+
+#### Standalone setup
+
+As this service is written in **Python 2.7**, it is recommended that all packages are
```
Comment (giovannicuriel) [inline README.md]: Just as a reminder: how about test history with Python 3.X and then set it as default? Python 2.7 will be deprecated next year (I think)

Comment (raulnegreiros) [reply README.md]: Yes, but we're planning to upgrade it in the near future.

File: README.md [hunk 6]
```diff
@@ -1,19 +1,52 @@
...
+
+In order to properly start History, the following variables must be set. All
+values are the default ones, and they need to be modified to the respective services' address.
+
+Key                         | Purpose                                                      | Default Value
+----------------------------|--------------------------------------------------------------|----------------
+HISTORY_DB_ADDRESS          |History database's address                                    |"mongodb"
+HISTORY_DB_PORT             |History database's port                                       |27017
+HISTORY_DB_REPLICA_SET      |History database's replica set address                        |None
+HISTORY_DB_DATA_EXPIRATION  |Seconds before removing an entry in MongoDB                   |604800
```
Comment (raulnegreiros) [inline README.md]: I think this variable is not available in this service, can you check it, please

**Pull request merged**